cms/djangoapps/contentstore/tests/test_courseware_index.py

```diff
@@ -1119,11 +1245,18 @@ def test_content_group_gets_indexed(self):
         )
 
         self.publish_item(self.store, self.html_unit1.location)
+        self.publish_item(self.store, self.split_test_unit.location)
 
         with patch(settings.SEARCH_ENGINE + '.index') as mock_index:
             self.reindex_course(self.store)
             self.assertTrue(mock_index.called)
             self.assertIn(self._html_group_result(self.html_unit1, [1]), mock_index.mock_calls)
+            self.assertIn(self._html_experiment_group_result(self.html_unit4, [0]), mock_index.mock_calls)
+            self.assertIn(self._html_experiment_group_result(self.html_unit5, [1]), mock_index.mock_calls)
+            self.assertIn(self._html_experiment_group_result(self.html_unit6, [2]), mock_index.mock_calls)
+            self.assertIn(self._vertical_experiment_group_result(self.condition_0_vertical, [0]), mock_index.mock_calls)
+            self.assertIn(self._vertical_experiment_group_result(self.condition_1_vertical, [1]), mock_index.mock_calls)
+            self.assertIn(self._vertical_experiment_group_result(self.condition_2_vertical, [2]), mock_index.mock_calls)
             mock_index.reset_mock()
 
     def test_content_group_not_assigned(self):
```

Comment:
I would like to see more tests here, mainly testing what happens if partition gets deleted and reindexed.